mojo-data/mojos/rulefit/stars/v1.00/rulefit_v1.00_stars_scores2.txt

```diff
@@ -0,0 +1,34 @@
+[328.4846820830261, 0.0]
+[430.56438055187266, 0.0]
+java.lang.ArrayIndexOutOfBoundsException
```

Comment:
predicting negative numbers [0,-1,-2,-3,-4,-5,-6,-7,-8] 3rd col is spectral type (categorical) and domain[-2] doesn't exist and that's the origin of java.lang.ArrayIndexOutOfBoundsException